pkg/controller/kibana/controller.go

```diff
@@ -240,5 +253,8 @@ type State struct {
 // NewState creates a new reconcile state based on the given request and Kibana resource with the resource
 // state reset to empty.
 func NewState(request reconcile.Request, kb *kbv1.Kibana) State {
-	return State{Request: request, Kibana: kb, originalKibana: kb.DeepCopy()}
+	newState := State{Request: request, originalKibana: kb.DeepCopy()}
+	newState.Kibana = newState.originalKibana.DeepCopy()
+	newState.Kibana.Status.ObservedGeneration = kb.Generation
+	return newState
```

Comment:
```suggestion
	newState := State{Request: request, originalKibana: kb.DeepCopy()}
	newState.Kibana = kb
	newState.Kibana.Status.ObservedGeneration = kb.Generation
	return newState
```

I thought this ^ would be sufficient, or am I missing something?

Reply:
I see what you were saying now.  ty.